src/Upgrader/Console/EvaluateConsole.php

```diff
@@ -106,4 +141,33 @@ public function execute(InputInterface $input, OutputInterface $output): int
 
         return Command::SUCCESS;
     }
+
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     *
+     * @throws \Upgrade\Application\Exception\UpgraderException
+     *
+     * @return array<\Codebase\Application\Dto\ModuleDto>
+     */
+    protected function getModuleList(InputInterface $input): array
```

Comment:
IMO it shouldn't be part of the console command.
Could be it part of DTO as it's not a business logic but just a data retrieving?

Reply:
Thanks 
I created OptionParser for option parsing logic